src/frontend/packages/auth-provider/src/index.js

```diff
@@ -32,5 +32,10 @@ export { default as useCheckPermissions } from './hooks/useCheckPermissions';
 export { default as usePermissionsWithRefetch } from './hooks/usePermissionsWithRefetch';
 export { default as useSignup } from './hooks/useSignup';
 
+// Password scoring
+export { default as PasswordStrengthIndicator } from './components/LocalLoginPage/PasswordStrengthIndicator';
+export { default as validatePasswordStrength } from './components/LocalLoginPage/validatePasswordStrength';
+export * as passwordScorer from './passwordScorer';
```

Comment:
I have an error `passwordScorer` not found when building the ActivityPods frontend. It works if I do an export like the other on this page. Maybe `export * as` is only available on some NodeJS versions ? (I use v16.17.1 currently) In which case, it's maybe better do like the others exports:
```js
export default {
  defaultScorer,
  createPasswordScorer,
  passwordStrength,
  defaultOptions
}
```